_health-care/_js/components/financial-assessment/ChildInformationSection.jsx

```diff
@@ -33,109 +41,113 @@ class ChildInformationSection extends React.Component {
           </div>
         </div>
 
-        <div className="row">
-          <div className="small-12 columns">
-            <p>Child's Name</p>
-            <FullName
-                value={this.props.data.childFullName}
-                onUserInput={(update) => {this.props.onStateChange('childFullName', update);}}/>
-          </div>
-        </div>
-
-        <div className="row">
-          <div className="small-12 columns">
-            <p>Child's relationship to you (check one)</p>
-            <ErrorableSelect label="Child’s relationship to you"
-                options={childRelationships}
-                value={this.props.data.childRelation}
-                onValueChange={(update) => {this.props.onStateChange('childRelation', update);}}/>
+        {this.props.data.hasChildrenToReport === true &&
+          <div>
+            <div className="row">
+              <div className="small-12 columns">
+                <p>Child&rsquo;s Name</p>
```

Comment:
I'm not 100% sure, but I'm pretty sure you don't need to escape the `’`. Since we're specifying the charset to be utf-8, you should only need to escape a few special reserved characters (e.g., `<`, `&`, etc.), of which `’` is not. You should be able to just type that character directly. This makes our markup a lot more readable. Maybe @crwallace can verify this is the case?